src/global_mapper/global_mapper_params.h

```diff
@@ -8,16 +8,32 @@ class Params {
  public:
   Params();
 
-  std::vector<double> voxel_xyz_min_;
-  std::vector<double> voxel_xyz_max_;
-  std::vector<double> voxel_resolution_;
+  // pixel map
+  double pixel_xy_min_[2];
+  double pixel_xy_max_[2];
```

Comment:
My interpretation of the GSG is that all member variables (in classes, not structs) have trailing underscores.

https://google.github.io/styleguide/cppguide.html#Variable_Names